tests/rpc-handler.test.ts

```diff
@@ -1,5 +1,7 @@
 import { JsonRpcProvider } from "@ethersproject/providers";
-import { HandlerConstructorConfig, RPCHandler, networkRpcs } from "../dist";
+import { networkRpcs } from "../dist";
```

Comment:
Is this unable to be imported from source?

P.S I felt testing the built package was better because that's the end product but I understand where you are coming from

Reply:
Fixed it. Importing from source now. 

> P.S I felt testing the built package was better because that's the end product but I understand where you are coming from

My understanding is that QA team performs blackbox tests who do not look into code. They would prefer a build. 
What we write are writing are progorammer tests. Even though the benefits are overlapping, the No.1 goal of programmer test is to provide quick feedback to progammer while working on the code. Programmer should be able to run tests frequently, and anything that slows down tests should be removed. Otherwise programmers will not run them as often as they should.  So a build step is a hurdle in this, but can be added if necessary.